doc/_static/drawer/mpldrawer_examples.py

```diff
@@ -0,0 +1,246 @@
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def box_gates(savefile="box_gates.png"):
+    drawer = MPLDrawer(n_wires=2, n_layers=1)
+
+    drawer.box_gate(layer=0, wires=(0, 1), text="CY")
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def box_gates_formatted(savefile="box_gates_formatted.png"):
+    box_options = {"facecolor": "lightcoral", "edgecolor": "maroon", "linewidth": 5}
+    text_options = {"fontsize": "xx-large", "color": "maroon"}
+
+    drawer = MPLDrawer(n_wires=2, n_layers=1)
+
+    drawer.box_gate(
+        layer=0, wires=(0, 1), text="CY", box_options=box_options, text_options=text_options
+    )
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def ctrl(savefile="ctrl.png"):
+    drawer = MPLDrawer(n_wires=2, n_layers=3)
+
+    drawer.ctrl(layer=0, wires=0, wires_target=1)
+    drawer.ctrl(layer=1, wires=(0, 1), control_values=[0, 1])
+
+    options = {"color": "indigo", "linewidth": 4}
+    drawer.ctrl(layer=2, wires=(0, 1), control_values=[1, 0], options=options)
+
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def CNOT(savefile="cnot.png"):
+    drawer = MPLDrawer(n_wires=2, n_layers=2)
+
+    drawer.CNOT(0, (0, 1))
+
+    options = {"color": "indigo", "linewidth": 4}
+    drawer.CNOT(1, (1, 0), options=options)
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def SWAP(savefile="SWAP.png"):
+    drawer = MPLDrawer(n_wires=2, n_layers=1)
+
+    drawer.SWAP(0, (0, 1))
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def SWAP_formatted(savefile="SWAP_formatted.png"):
+    swap_keywords = {"linewidth": 2, "color": "indigo"}
+
+    drawer = MPLDrawer(n_wires=2, n_layers=1)
+
+    drawer.SWAP(0, (0, 1), options=swap_keywords)
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def measure(savefile="measure.png"):
+    drawer = MPLDrawer(n_wires=1, n_layers=1)
+
+    drawer.measure(0, 0)
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def measure_formatted(savefile="measure_formatted.png"):
+    drawer = MPLDrawer(n_wires=1, n_layers=1)
+
+    measure_box = {"facecolor": "white", "edgecolor": "indigo"}
+    measure_lines = {"edgecolor": "indigo", "facecolor": "plum", "linewidth": 2}
+    drawer.measure(0, 0, box_options=measure_box, lines_options=measure_lines)
+    plt.savefig(folder / savefile)
+    plt.close()
+
+
+def integration(style="default", savefile="example_basic.png"):
+    plt.style.use(style)
+    drawer = MPLDrawer(n_wires=5, n_layers=5)
+
+    drawer.label(["0", "a", r"$|\Psi\rangle$", r"$|\theta\rangle$", "aux"])
+
+    drawer.box_gate(0, [0, 1, 2, 3, 4], "Entangling Layers",
+        text_options={"rotation": "vertical"})
+    drawer.box_gate(1, [0, 1], "U(θ)")
+
+    drawer.box_gate(1, 4, "Z")
+
+    drawer.SWAP(1, (2, 3))
+    drawer.CNOT(2, (0, 2))
+
+    drawer.ctrl(3, [1, 3], control_values=[True, False])
+    drawer.box_gate(layer=3, wires=2, text="H", box_options={'zorder': 4},
+        text_options={'zorder': 5})
```

Comment:
If not providing a `zorder` value, does it simply draw the box gate on top of everything else? I guess in most cases a user would simply draw things in order, and leave `zorder` be?